CHANGELOG.asciidoc

```diff
@@ -41,6 +41,7 @@ communication - {pull}2996[#2996]
 * Add the <<config-long-field-max-length>> config to enable capturing larger values for specific fields - {pull}3027[#3027]
 * Provide fallback correlation when `ecs-logging-java` is used - {pull}3064[#3064]
 * Added separate Java 8 build with updated log4j2 - {pull}3076[#3076]
+* Added tests for Quarkus / RestEasy, adjusted vert.x router transaction name priority - {pull}1765[#1765]
```

Comment:
This will need to be updated as there was a release since this PR was last updated.